src/InfoPanel/InfoPanel.jsx

```diff
@@ -1,45 +1,79 @@
 import classnames from "classnames";
-import React, {PropTypes} from "react";
+import React, {Component, PropTypes} from "react";
+import FontAwesome from "react-fontawesome";
+import ReactCollapsible from "react-collapsible";
 
 import "./InfoPanel.less";
 
 
 /**
- * Base presentational component for the displaying information in  paneled format.
+ * Base presentational component for the displaying information in paneled format.
  */
-export default function InfoPanel({children, className, title, footer}) {
-  const {cssClass} = InfoPanel;
-
-  return (
-    <div className={classnames(cssClass.CONTAINER, className)}>
-      <div className={cssClass.HEADER}>
-        <h4 className={cssClass.TITLE}>
-          {title}
-        </h4>
-      </div>
-      <div className={cssClass.CONTENT}>
-        {children}
+export default class InfoPanel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isCollapsed: props.defaultOpen,
+    };
+  }
+
+  render() {
+    const {children, className, title, footer, collapsible, defaultOpen} = this.props;
+    const {isCollapsed} = this.state;
+    const {cssClass} = InfoPanel;
+    let collapseArrow = "";
+
+    if (collapsible && isCollapsed) {
+      collapseArrow = <FontAwesome name="caret-down" />;
+    } else if (collapsible && !isCollapsed) {
+      collapseArrow = <FontAwesome name="caret-right" />;
+    }
+
+    return (
+      <div className={classnames(cssClass.CONTAINER, className)}>
+        <ReactCollapsible trigger={collapsible ?
+          (<div>
+            <div className={cssClass.COLLAPSE_ARROW}>{collapseArrow}</div>
```

Comment:
The collapse toggle is inaccessible as a div without aria roles.

I'd recommend taking a look at accessible accordions to model after like this one: https://github.com/springload/react-accessible-accordion#accessibility

Reply:
Thanks for the find, Carl!  I'm looking into this with Vicky and wondering if we get keyboard accessibility and the aria tags/role set up correctly, think we're fine to use `react-collapsible`?  Or do you think we should switch to something that was built with accessibility in mind, ie `react-accessible-accordion`?

Reply:
We would have to test it to make sure, I'm uncertain as to how react-collapsible deals with accessibility.